### PR TITLE
Login

### PR DESCRIPTION
**Pull request by fionamei**

### CODE REVIEW

File: ComfortZoneApp/app/src/main/java/com/example/comfortzone/models/ComfortLevelEntry.java [hunk 1]
```diff
@@ -0,0 +1,43 @@
...
+import com.parse.ParseClassName;
+import com.parse.ParseObject;
+import com.parse.ParseUser;
+
+@ParseClassName("ComfortLevelEntry")
+public class ComfortLevelEntry extends ParseObject {
+
+    public static final String KEY_TEMP = "temp";
+    public static final String KEY_COMFORTLEVEL = "comfortLevel";
+    public static final String KEY_USER = "user";
```
Comment (kimjaspermui) [inline ComfortZoneApp/app/src/main/java/com/example/comfortzone/models/ComfortLevelEntry.java:13]: nice!

File: ComfortZoneApp/app/src/main/res/layout/activity_initial_comfort_actvitiy.xml [hunk 2]
```diff
@@ -14,76 +14,93 @@
...
         android:id="@+id/textView3"
         android:layout_width="match_parent"
         android:layout_height="wrap_content"
         android:layout_marginHorizontal="16dp"
-        android:gravity="center"
-        android:layout_marginTop="16dp"
-        android:text="A '0' represents the point where you are crazy cold! Similarly, a '10' represents a crazy hot degree. And '5' is your estimated perfect temperature!"
+        android:textColor="@color/black"
+        android:layout_marginVertical="16dp"
+        android:text="ComfortZone will track your comfort zones for different temperature levels! A '0' represents the point where you are crazy cold! Similarly, a '10' represents a crazy hot degree. And '5' is your estimated perfect temperature!"
```
Comment (kimjaspermui) [inline ComfortZoneApp/app/src/main/res/layout/activity_initial_comfort_actvitiy.xml:29]: let's put strings in strings.xml and manage them there

File: ComfortZoneApp/app/src/main/res/layout/activity_initial_comfort_actvitiy.xml [hunk 3]
```diff
@@ -14,76 +14,93 @@
         android:layout_height="wrap_content"
         android:layout_marginHorizontal="16dp"
         android:layout_marginTop="40dp"
-        android:gravity="center"
-        android:text="Hello there! Welcome to ComfortZone, an app focused on tracking your daily comfort zones. "
-        android:textSize="20sp" />
+        android:textStyle="bold"
+        android:text="Welcome!"
+        android:textColor="@color/black"
+        android:textSize="35sp" />
```
Comment (kimjaspermui) [inline ComfortZoneApp/app/src/main/res/layout/activity_initial_comfort_actvitiy.xml:20]: ideally, you also want to put dimensions/sizes/paddings/margins/any numbers you add to dimens.xml but it's less common afaik, but let's move it there as well, see how: https://stackoverflow.com/questions/29128178/how-to-use-dimens-xml-in-android

**Review by kimjaspermui — APPROVED** [review]
looks good over all, no major issues, we can merge this to master and forward fix the feedback. Let's talk about amending commits in our 1:1 - this will help organize your commits more for easy review

File: ComfortZoneApp/app/src/main/res/layout/input_layout.xml [hunk 4]
```diff
@@ -37,11 +44,11 @@
         android:layout_width="match_parent"
         android:layout_height="wrap_content"
         android:layout_marginBottom="8dp"
+        app:boxBackgroundColor="@color/grey"
```
Comment (varhalemonica) [inline ComfortZoneApp/app/src/main/res/layout/input_layout.xml:47]: Do we need to add an id for this component TextInputLayout as well? Generally it's a good practice to add id to all components.

File: ComfortZoneApp/app/src/main/java/com/example/comfortzone/InitialComfortActvitiy.java [hunk 5]
```diff
@@ -71,26 +74,33 @@ public void onClick(View v) {
...
-                addEntry(entryZero);
-                addEntry(entryFive);
-                addEntry(entryTen);
-            }
-        });
-
-
 
-    }
+        entryZero = new ComfortLevelEntry(user, tempzero, 0);
```
Comment (varhalemonica) [inline ComfortZoneApp/app/src/main/java/com/example/comfortzone/InitialComfortActvitiy.java]: Can you initialize 0, 5, 10 as private static constants in this class? You can name it as START_COMFOR_LEVEL, MID_COMFORT_LEVEL, END_COMFORT_LEVEL

**Pull request merged**